ceph-releases/jewel/ubuntu/14.04/demo/Dockerfile

```diff
@@ -7,8 +7,17 @@
 FROM ceph/base:tag-build-master-jewel-ubuntu-14.04
 MAINTAINER Sébastien Han "seb@redhat.com"
 
+# Install s3cmd to support creation of buckets
+RUN ["apt-get", "update", "-y"]
```

Comment:
we never used brackets in the repository, just for consistency purpose could you please re-use the following schema? https://github.com/ceph/ceph-docker/blob/master/ceph-releases/jewel/ubuntu/14.04/base/Dockerfile

is it a best practice btw?